bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_10/A5_10_15.java

```diff
@@ -22,7 +25,27 @@
 @NonNullByDefault
 public class A5_10_15 extends A5_10 {
 
+    private final double maxScaledTemp = 41.2;
+    private final double minScaledTemp = -10.0;
```

Comment:
I guess they could be static?
```suggestion
    private static final double MAX_SCALED_TEMP = 41.2;
    private static final double MIN_SCALED_TEMP = -10.0;
```

Reply:
Of course🤦‍♂️  coming from c# where a const is always static. However I changed the calculation and think that these consts are not neccessary anymore.

Reply:
Yeah, in Java there is no `const` or `readonly` like in C#. `final` is similar to `readonly`, but assignment can also happen outside the constructor. Add `final` and we are pretty close to a `const`. 🙂